etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/base/ETLBase.java

```diff
@@ -94,39 +92,25 @@ protected Connection getHiveJDBCConnection(String dbName){
         }
 
     }
-    HiveMetaStoreClient hclient =null;
+    HiveMetaStoreClient hiveClient =null;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1213)